benchmark/worker/benchmark_client.go

```diff
@@ -246,7 +253,7 @@ func startBenchmarkClient(config *testpb.ClientConfig) (*benchmarkClient, error)
 	return bc, nil
 }
 
-func (bc *benchmarkClient) doCloseLoopUnary(conns []*grpc.ClientConn, rpcCountPerConn int, reqSize int, respSize int) {
+func (bc *benchmarkClient) doCloseLoopUnary(conns []*grpc.ClientConn, rpcCountPerConn int, reqSize int, respSize int, poissonLambda *float64) {
```

Comment:
This seems like [control coupling](https://en.wikipedia.org/wiki/Coupling_(computer_programming)#:~:text=tools%20and%20devices.-,Control%20coupling,-Control%20coupling%20is) to me. I think you probably want two methods: `doCloseLoopUnary` and `doOpenLoopUnary`.

Reply:
Discussed offline; let's keep as is for clean scalability of the old code in the codebase.